interfaces/repo_test.go

```diff
@@ -123,11 +127,18 @@ func (s *RepositorySuite) TearDownTest(c *C) {
 	s.BaseTest.TearDownTest(c)
 }
 
-func addPlugsSlots(c *C, repo *Repository, yamls ...string) []*snap.Info {
-	result := make([]*snap.Info, len(yamls))
-	for i, yaml := range yamls {
-		info := snaptest.MockInfo(c, yaml, nil)
-		result[i] = info
+func addPlugsSlotsFromInstances(c *C, repo *Repository, instanceNamesAndYamls ...string) []*snap.Info {
+	c.Assert(len(instanceNamesAndYamls)%2, Equals, 0, Commentf("unexpected length"))
```

Comment:
"unexpected length: an even number expected" or "unexpected length: expected instance name, yaml pairs" would make the intention clear IMHO.